recipes-core/image/image.bb

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "A small image just capable of allowing a device to boot."
```

Comment:
if "image.bb" will become a build target - it looks not good and another name better to select

Reply:
You can take 
https://github.com/aoscloud/meta-aos-rcar-gen4/blob/main/aos-rcar-gen4.yaml
as reference for image structure

Reply:
Ok

Reply:
rpi5-image-minimal-domd.bb